manifest.json

```diff
@@ -291,9 +291,9 @@
         }
       },
       "select2": {
-        "version": "4.0.3",
+        "version": "4.0.13",
```

Comment:
You will also need to change the path in core/templates/third-party-imports/select2.import.ts in order to get this working.

Reply:
Actually, this change by mistake came here. This is of another issue actually. I will remove it in the next commit.